### PR TITLE
fix: disable published items and collections

### LINKED ISSUE #1114: name field is not disabled on published items

### PR DESCRIPTION
**Pull request by cazala**
Fixes #1114

### CODE REVIEW

**Comment by nicosantangelo** [comment]
Ready to rebase!

**Comment by cazala** [comment]
@nicosantangelo rebased :)

File: src/components/ItemEditorPage/RightPanel/Input/Input.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class Input extends React.PureComponent<Props, State> {
     const { label, disabled } = this.props
 
     return (
-      <div className="Input">
+      <div className={`Input ${disabled ? 'is-disabled' : ''}`.trim()}>
```
Comment (nicosantangelo) [inline src/components/ItemEditorPage/RightPanel/Input/Input.tsx:34]: cheeky trim

**Review by nicosantangelo — APPROVED** [review]
LGTM

**Pull request merged**